### DISCUSSION TITLE
Survey for Women

### DISCUSSION THREAD

**Discussion by rjheby** [PullDating/Org#121] [General]
What kind of questions do we want to ask women?

For me a few that come up as a man:

- do you feel you have more matches than you can deal with on dating apps?
- how appealing is this feature (1-10) having a limited number of high quality matches versus unlimited matches?
- what are the main reasons you wouldn't reply to a match? (Boring, generic, not interested, too many matches, just browsing)
- how much do you agree with this statement (1-10) all the good guys are already taken?
- how much do you agree with this statement (1-10) safety is a concern for me on dating apps
- how willing would you be to provide a representation of your physique on a dating app?
- demographics: gender, age range, sexual orientation

**Comment by WilliamDormer** [comment]
I think this type of survey needs to be the next priority for the marking people in our project.
